virttest/test_setup.py

```diff
@@ -1342,10 +1349,22 @@ def set_vf(self, pci_pf, vf_no="0"):
         :params vf_no: Number of Vfs to be virtualized
         :return: True on success, Fail on failure
         """
-        cmd = "echo %s > /sys/class/net/%s/device/sriov_numvfs" % (vf_no, pci_pf)
+        if ARCH == 'ppc64le':
+            path = "/sys/class/net/%s/device/sriov_numvfs" % pci_pf
+        elif ARCH == 'x86_64':
+            path = "/sys/bus/pci/devices/%s/sriov_numvfs" % pci_pf
```

Comment:
As I just checked, the second path exists on both arches. Update to use the second one. Thanks!